### ISSUE TITLE
question about attention_mask and pad_token_id

### ISSUE THREAD

**Issue by zxilin01** [AI-secure/aug-pe#5]
1. when i run generate.sh of openreview, it outputs:

The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.

Is this right? Or I need to modify any parameter? 

2. When running generate.sh of openreview, I found that the output of each epoch was like below. Why are there so many instances of 0.0? Is this normal? And the FID is also very large.

![Image](https://github.com/user-attachments/assets/605f18ad-96c7-46c4-98e8-7883bf8a5fa7)

![Image](https://github.com/user-attachments/assets/dd5b29b8-ecf2-4880-92aa-ce44cf4d5ef7)

If I could receive your reply, I would greatly appreciate it！
